src/components/swapv2/SwapModalFooter.tsx

```diff
@@ -170,24 +177,36 @@ export default function SwapModalFooter({
           {veryHighPriceImpact ? <Trans>Price impact is Very High!</Trans> : <Trans>Price impact is High!</Trans>}
         </AutoRow>
       )}
+      <HurryUpBanner startedTime={startedTime} />
```

Comment:
LOL. still show this on confirm modal? Let remove it and show it on waiting modal only. I will combat with Kami if need

Reply:
You might want to join Solana slack. There was a long thread about it